README.md

```diff
@@ -5,42 +5,116 @@
 [![Dependencies][gemnasium-image]][gemnasium-url]
 [![Widgets][widget-count-image]][widget-count-url]
 
-Mozaïk is a tool based on nodejs / react / d3 / stylus to easily craft beautiful dashboards.
-It ships with several useful [widgets](https://github.com/plouc/mozaik/wiki/widgets) (for now, focused on CI and monitoring) such as Jenkins, Travis, Sensu…
-You can switch between 5 available themes or [make your own one](https://github.com/plouc/mozaik/wiki/theming).
-One of its handy feature is to be able to make a *rotation* between several dashboards with a single [config file](https://github.com/plouc/mozaik/wiki/configuration).
+Mozaïk is a tool based on nodejs / react / reflux / d3 / stylus to easily craft beautiful dashboards. [See demo](http://mozaik.herokuapp.com/)
 
-![preview](https://raw.githubusercontent.com/wiki/plouc/mozaik/assets/mozaik-panorama.png)
+![preview](https://raw.githubusercontent.com/juhamust/mozaik/readme/preview.png)
```

Comment:
I would have included the screenshot in PR, but to be honest, I don't know how to do that with wiki.